spec/features/inviting_new_submissions_in_place_of_expired_ones_feature_spec.rb

```diff
@@ -0,0 +1,40 @@
+# Testing rake tasks method found at:
+# http://carlosplusplus.github.io/blog/2014/02/01/testing-rake-tasks-with-rspec/
+# https://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+
+require 'rails_helper'
+require 'rake'
+
+describe 'inviting new submissions in place of expired ones' do
```

Comment:
So you do all this stuff, 30+ lines of code, with a setup that is tightly coupled to rails and code that is not yours, just to check if it calls the correct service? And you have a spec for that service as well? This seems like a huge test overkill to me. I know how you like to test things thoroughly, but just because you can test something, doesn't mean you should ;)